index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+<head>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css" integrity="sha384-9gVQ4dYFwwWSjIDZnLEWnxCjeSWFphJiwGPXr1jddIhOegiu1FwO5qRGvFXOdJZ4" crossorigin="anonymous">
+  <link rel="stylesheet" href="./css/main.css">
+</head>
+<body>
+<div class="container-fluid">
+  <article>
+
+  <!-- header-->
+    <section class="header">
+      <!-- navigation bar -->
+      <nav class="navbar navbar-expand-lg">
+        <a class="navbar-brand" href="#">
+          <div class="logo"></div>
+        </a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarNav">
+          <ul class="navbar-nav">
+            <li class="nav-item">
+              <a class="nav-link" href="#">Sponsor</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Branding</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Contact Us</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Code of Conduct</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+      <!-- introduction -->
+      <div class="intro">
+        <div class="heading text-center">
+          <h1> Thanks for Technica 2019! <br> Stay tuned for this year </h1>
```

Comment:
Are we considering adding translations to our site content? If so, we should implement a translation pattern later (def not now). I will flag w/ the execs...